binary_transparency/firmware/integration/ft_test.sh

```diff
@@ -92,27 +92,43 @@ go run ../cmd/emulator/dummy/ \
     ${COMMON_FLAGS}
 
 ####################
-banner "Fiddle with installed firmware and try to boot device"
+banner "Replace installed firmware and try to boot device"
 cp -v ../testdata/firmware/dummy_device/hacked.wasm ${DEVICE_STATE}/firmware.bin
 EXPECT_FAIL "firmware measurement does not match" \
     go run ../cmd/emulator/dummy/ \
         --dummy_storage_dir="${DEVICE_STATE}" \
         ${COMMON_FLAGS}
 
 ####################
-banner "Fiddle with installed firmware & manifest and try to boot device"
+banner "Replace installed firmware, update manifest hash, and try to boot device"
 HACKED_FIRMWARE=../testdata/firmware/dummy_device/hacked.wasm
 cp -v ${HACKED_FIRMWARE} ${DEVICE_STATE}/firmware.bin
 malware_hash=$(sha512sum ${DEVICE_STATE}/firmware.bin | awk '{print $1}' | xxd -r -p | base64 -w 0 )
 echo "new hash ${malware_hash}"
 mv ${DEVICE_STATE}/bundle.json ${DEVICE_STATE}/bundle.json.orig
 # This beastly jq command unpacks the nested json structures and replaces just the fimware measurement with the one from our hacked firmware:
 jq --arg hacked ${malware_hash} -c '.ManifestStatement=(.ManifestStatement|@base64d|fromjson|.Metadata=(.Metadata|@base64d|fromjson|.ExpectedFirmwareMeasurement=$hacked|tojson|@base64)|tojson|@base64)' ${DEVICE_STATE}/bundle.json.orig > ${DEVICE_STATE}/bundle.json
```

Comment:
We can replace this with the new command and then drop the requirement on `jq` now?

Reply:
I thought about that and stopped short for "narrative" reasons ("I'll try this... bah", "ok, I'll up my game/steal keys/build tooling", etc. etc.), but on reflection you're right - the shell and jq stuff is pretty opaque, and I can pop the narrative into the comments.